contracts/vaults/BaseVaultWithSwap/base/RibbonVault.sol

```diff
@@ -731,10 +720,16 @@ contract RibbonVault is
      * @return total balance of the vault, including the amounts locked in third party protocols
      */
     function totalBalance() public view returns (uint256) {
+        // After calling closeRound, current option is set to none
```

Comment:
Instead of doing this check, why don't we just set the `lockedAmount` to 0?

Reply:
We need to fix the `lockedAmount` at `closeRound` since we mint new depositor shares there. This is so `rollToNextOption` can read from `lockedAmount` and create the correct amount of short position. Alternatively, we can mint more shares at `rollToNextOption` and re-calculate `lockedAmount`